<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Edition</class>
 <widget class="QWidget" name="Edition">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>590</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="5">
    <widget class="QPushButton" name="butSave">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="butLoad">
     <property name="text">
      <string>Load</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QToolButton" name="butNew">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>New</string>
     </property>
     <property name="popupMode">
      <enum>QToolButton::MenuButtonPopup</enum>
     </property>
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonTextOnly</enum>
     </property>
     <property name="arrowType">
      <enum>Qt::DownArrow</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="5">
    <widget class="QTreeView" name="treeView"/>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>To modify a field, click on 'value' column</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>butSave</sender>
   <signal>clicked()</signal>
   <receiver>Edition</receiver>
   <slot>actionSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>379</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>butLoad</sender>
   <signal>clicked()</signal>
   <receiver>Edition</receiver>
   <slot>actionLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>actionSave()</slot>
  <slot>actionLoad()</slot>
 </slots>
</ui>
